geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/geoportal/vars.yaml

```diff
@@ -81,7 +81,7 @@ vars:
             maxTilesLoading: 128
           view:
             <<: *view
-            constrainResolution: True
+            constrainResolution: False
```

Comment:
Can we add a small comment in this file to explain why "False" is better?
So we don't make the mistake again of changing this setting?

Reply:
It's not better, just... different. It's a pure Openlayers feature.
Well, I documented the commit, and the PR, + two issues and one jira ticket, that's already something.
But we could add this documentation in ngeo directly too https://camptocamp.github.io/ngeo/master/apidoc/types/srcapi_store_config.View.html#__type.constrainResolution  or eventually here:
https://camptocamp.github.io/c2cgeoportal/master/integrator/ngeo.html#view